### DISCUSSION TITLE
q-select/q-item -> not reactive when list item is visible

### DISCUSSION THREAD

**Discussion by nvima** [quasarframework/quasar#11806] [General - Components / Directives / etc] [answered]
Hello,
i currently have a little problem with my q-select.
when i add an entry, the list expands as expected. But if I have the list open and add an entry, the change is only displayed after closing and reopening it.
I'm still pretty new to vue and quasar. And can't really tell if the problem is in Vue or Quasar, or me: D
I hope someone here can give me a tip on how to proceed.

https://codepen.io/jens3911-the-lessful/pen/abLEajr

Edit:
It seems like some reactivity is there, because the entries in the list are exchanged when it is open, but the list is not enlarged

**Comment by MilosPaunovic** [comment]
You were missing `value` prop for `QSelect`.
[https://codepen.io/milospaunovic/pen/VwMyEem?editors=1111](https://codepen.io/milospaunovic/pen/VwMyEem?editors=1111)

**Reply by nvima** [reply]
> You were missing `value` prop for `QSelect`. https://codepen.io/milospaunovic/pen/VwMyEem?editors=1111

I didn't mean that, sorry if I got it wrong. I'm interested in the list of select items that can be clicked up. If I add an entry with the +, the list is expanded, but not if I opened it while adding

**Comment by pdanpdan** [comment, marked as answer]
The list of options is only processed on opening the popup.
For updating it manually you have a `refresh` method on QSelect instance.
https://codepen.io/pdanpdan/pen/WNZdaQK?editors=1010

**Reply by nvima** [reply]
thanks for the fast response :)
works great
